umbraco-workflow/upgrading-workflow.md

```diff
@@ -0,0 +1,74 @@
+# Upgrading Umbraco Workflow
+
+This article shows how to manually upgrade Umbraco Workflow to run the latest version. Umbraco Workflow displays a prompt in the **Workflow** section when a new version is available.
+
+{% hint style="info" %}
+If you are migrating from Plumber to Umbraco Workflow, see the [Migration from Plumber to Umbraco Workflow](#migration-from-plumber-to-umbraco-workflow) section.
+{% endhint %}
+
+## Get the latest version of Umbraco Workflow
+
+To get the latest version of Umbraco Workflow, you can upgrade using either of the two options:
+
+- [NuGet](#nuget)
+- [Visual Studio](#visual-studio)
+
+### NuGet
+
+NuGet installs the latest version of the package when you use the `dotnet add package Umbraco.Workflow` command unless you specify a package version:
+
+  `dotnet add package Umbraco.Workflow --version <VERSION>`
+
+After you have added a package reference to your project by executing the `dotnet add package Umbraco.Workflow` command in the directory that contains your project file, run `dotnet restore` to install the package.
```

Comment:
The command that needs to be run is mentioned 3 times in this section 🙈 
I've tried to rephrase the final part a bit, to get rid of the third time its mentioned.

Let me know what you think!

```suggestion
Run the command listed above in the directory that contains your project file. After that, run `dotnet restore` to install the package.
```